internal/local/command_kafka_broker_describe.go

```diff
@@ -0,0 +1,83 @@
+package local
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/pkg/broker"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/utils"
+)
+
+func (c *Command) newKafkaBrokerDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe [id]",
+		Short: "Describe a local Kafka broker.",
+		Long:  "Describe cluster-wide or per-broker configuration values.",
```

Comment:
I think we should align with the future state of the `broker` commands, that is:
1. This command should strictly be for printing `broker` configurations, not cluster-wide configurations (remove the `--all` flag)
2. Cluster-wide configurations should be printed with `confluent local kafka cluster configuration list`.

I was waiting for v4 to make this change, but seems like we have an opportunity to make it early here.

Reply:
sounds good. Adding this change.